utils/common/create_chaos_pod.yml

```diff
@@ -0,0 +1,44 @@
+- block:
+
+    - name: Creating the chaos pod in {{ pod_ns }} namespace
+      shell: >
+        kubectl apply -f {{ c_path }} -n {{ pod_ns }}
+      args:
+        executable: /bin/bash
+      register: pumba_pod_result
+
+    - name: Check for running status of the chaos pod
+      shell: >
+        kubectl get pods -l name=pumba-sig-kill-{{ run_id }} -n {{ pod_ns }} -o custom-columns=:.status.phase --no-headers
+      register: pod_result
+      until: "((pod_result.stdout.split()|unique)|length) == 1 and 'Running' in pod_result.stdout"
+      delay: 2
+      retries: 90
+      ignore_errors: true
+
+    - block:
+
+        - name: Delete the chaos pod, If not comming to running state
```

Comment:
Delete the pumba pod, if it isn't scheduled successfully

Reply:
I am using chaos pod to make it generic, as this util will be used by other experiments too.